packages/account/src/Sections/Security/AccountLimits/account-limits.jsx

```diff
@@ -179,7 +179,11 @@ class AccountLimits extends React.Component {
                                         <Row>
                                             <TableHeader>
                                                 <div className='account-management-flex-wrapper'>
-                                                    <TextComp className='account-limit-container-title'>
+                                                    <TextComp
```

Comment:
Still using an alias?

Reply:
I asked you Salar :) if you remember on slack, and you told me it’s better Ali change the Text component from `Components/text`
Ali's PR is not merged to the master yet. After that I can remove the alias.

Reply:
OK OK :))